### DISCUSSION TITLE
Error Forbiden, 403 - id: "security_csrf_violation"

### DISCUSSION THREAD

**Discussion by pedrodgoes** [ory/kratos#3517] [Q&A]
I'm putting my project into a production environment, using netlify, but I'm getting this error when I try to log in. I really don't know what could be going on, I've read the documentation, I've seen various forums about it but I've yet to see a solution, at least not one that solved my case. Thank you if anyone can help, and if you have any questions about my problem, just ask.

My application url: https://netlify.myftp.org/
My api url: https://orytesla.myftp.org/

Action: I enter my login and password, then click Log In and get this error in the console.

![image](https://github.com/ory/kratos/assets/31218525/6d01ea52-f586-4e87-b62d-87289fb0b54a)

My Headers
![image](https://github.com/ory/kratos/assets/31218525/ef732183-95c2-464f-953f-f681fa22a3a9)

My Payload
![image](https://github.com/ory/kratos/assets/31218525/520e9aba-b6c2-4954-81f7-35fe4d400d05)

My Preview
![image](https://github.com/ory/kratos/assets/31218525/5c6b9d70-9739-4b9c-b667-01f1a55b1f4e)

My cookies, because apparently it has something to do with it, and I'm posting a picture of it too.
![image](https://github.com/ory/kratos/assets/31218525/af679409-ab12-4489-af83-c2ab129ee491)

**Comment by aeneasr** [comment]
Dupe https://github.com/ory/network/discussions/338
